### PR TITLE
[Typst] Update to v0.7.0

### PR DESCRIPTION
**Pull request by jakobjpeters**

### CODE REVIEW

**Comment by jakobjpeters** [comment]
```
error: package `typst-syntax v0.7.0 (/workspace/srcdir/typst/crates/typst-syntax)` cannot be built because it requires rustc 1.70 or newer, while the currently active rustc version is 1.69.0
```

File: T/Typst/build_tarballs.jl [hunk 1]
```diff
@@ -20,7 +20,9 @@ install_license LICENSE
 
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
-platforms = filter(p -> !(Sys.iswindows(p) && arch(p) == "i686"), supported_platforms())
+platforms = filter(supported_platforms()) do p
+    !(Sys.iswindows(p) && arch(p) == "i686" || Sys.islinux(p) && arch(p) == "powerpc64le")
```
Comment (giordano) [inline T/Typst/build_tarballs.jl:24]:
```suggestion
    !((Sys.iswindows(p) && arch(p) == "i686") || (arch(p) == "powerpc64le"))
```

**Comment by giordano** [comment]
Thanks!  And sorry for the long wait!

**Comment by jakobjpeters** [comment]
> Thanks! And sorry for the long wait!

No worries! I've seen you posting about LLVM

**Pull request merged**